docs/rules/attribute-hyphenation.md

```diff
@@ -35,7 +36,7 @@ This rule enforces using hyphenated attribute names on custom components in Vue
 ```json
 {
   "vue/attribute-hyphenation": ["error", "always" | "never", {
-    "ignore": []
+    "ignore": [],
   }]
```

Comment:
Something is meesd up here now. Please change it back to how it was before.

Reply:
Sorry about that! Looks like my text editor got a little excited about linting.